Documentation/allow_legacy_mode.md

```diff
@@ -0,0 +1,101 @@
+## Allow-legacy mode
+
+etcd v2.0 uses internal API v2 protocol, which is not compatible with the v1 protocol.
```

Comment:
Allow-legacy should allow your etcd cluster to have a minimal amount of downtime -- less than 1 minute for clusters storing less than 50 MB.